packages/@okta/vuepress-theme-prose/components/TermsAndConditionsDialog.vue

```diff
@@ -0,0 +1,267 @@
+<template>
+  <DialogWindow
+    @close="closeTermsConditionsDialog()"
+    title="Tell us more about yourself"
+  >
+    <div class="dialog-text">
+      <p>
+        The information you provide will only be used to help us improve your
+        experience. Your personal data is safe and secure with us.
+      </p>
+      <label for="tac-country">
+        Country
+        <select
+          name="Country"
+          id="tac-country"
+          class="tac-row"
+          ref="countryEl"
+          v-model="selectedCountry"
+          @change="
+            region = $event.target.value;
+            resetFields();
+          "
+        >
+          <option disabled selected></option>
+          <option
+            v-for="country in countrieList"
+            v-bind:key="country.value"
+            :value="country.value"
+            >{{ country.name }}</option
+          >
+        </select>
+      </label>
+      <label for="tac-regionData" v-show="region.list.length">
+        {{ region.type }}
+        <select
+          name="Satate"
```

Comment:
```suggestion
          name="State"
```

Reply:
hhmm probably will be a good idea to use `region` instead, this field could be `state` or `province`